internal/host/common.go

```diff
@@ -36,7 +36,7 @@ const (
 	statusInfoInstallationInProgressWritingImageToDiskTimedOut = "Host failed to install because its installation stage $STAGE did not sufficiently progress in the last $MAX_TIME."
 	statusInfoHostReadyToBeBound                               = "Host is ready to be bound to a cluster"
 	statusInfoBinding                                          = "Host is waiting to be bound to the cluster"
-	statusRebootTimeout                                        = "Host failed to reboot within timeout, please boot the host from the the OpenShift installation disk $INSTALLATION_DISK. The installation will resume once the host has rebooted"
+	statusRebootTimeout                                        = "Host failed to pull ignition within timeout, please check the host console and verify the it boot from the the OpenShift installation disk $INSTALLATION_DISK and has network access to the cluster API. The installation will resume once the host successfully boots"
```

Comment:
you need to change https://github.com/openshift/assisted-test-infra/blob/6aaf9a3f08d7e9f17ff281c8cfdf29f55a8a5b9d/src/consts/consts.py#L290
(preferably support the two formats for backwards/forwards compatibility)

Reply:
Thanks, let's agree on the phrasing and I'll update assisted-test-infra

Reply:
The update to assisted-test-infra merged